src/demo-app/a11y/dialog/dialog-neptune-iframe-a11y.html

```diff
@@ -0,0 +1,12 @@
+<h2 md-dialog-title>Neptune</h2>
```

Comment:
That doesn't give a way to focus the `role="dialog"` element, though, which I _think_ is what we'd want (but I'm not 100% sure).